### PR TITLE
Update environment.yml

### PR DESCRIPTION
**Pull request by jlaura**
Updates the environment to use the new plio release with PVL >= 1.0

Depending upon CI to test this. Will mark as a draft until I have passing CI.

### CODE REVIEW

**Comment by jlaura** [comment]
The environment is properly setting up and the tests are passing as expected (2 xfails - intentional fails). The CI failure is because of the code coverage upload issue (#557)

File: environment.yml [hunk 1]
```diff
@@ -22,15 +22,15 @@ dependencies:
...
+  - plio >= 1.2.5
   - plurmy
   - psycopg2
   - proj >= 7.0.0, <8
   - pyproj
   - pytest
   - pytest-cov
   - pysis
-  - pvl < 1.0
+  - pvl >= 1.0
```
Comment (jessemapel) [inline environment.yml:33]: Does AutoCNet actually use PVL 1.0 features or is this just a transitive dependency from PLIO?

Comment (jlaura) [reply environment.yml:33]: Transitive - so probably safe to remove from here now that we aren't trying to do weird version mixing things. Ohh -this needs to update the meta.yml too! PR inbound with both changes.

File: conda/meta.yaml [hunk 2]
```diff
@@ -43,27 +43,27 @@ requirements:
...
       - pyyaml
       - redis-py
-      - scikit-image
+      - scikit-image>=0.17
       - scikit-learn
       - scipy<=1.2.1
       - shapely
-      - sqlalchemy
-      - sqlalchemy-utils
+      - sqlalchemy>-1.4,<1.5
```
Comment (acpaquette) [inline conda/meta.yaml:65]: What does the `>-` mean? Or is it a typo?

**Review by acpaquette — COMMENTED** [review]
The plio and pvl changes look in line with the updates. One small question but the other changes look good!

**Comment by rbeyer** [comment]
If *this* package has `import pvl` in its code (which it does), then shouldn't the line for `pvl` in these files just be updated and not deleted?  Otherwise you're relying on other dependencies to install *pvl* for you.  Which they will, but seems strange to me.

**Pull request closed without merging**